[2025-10-02T11:34:18.218+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T11:34:18.233+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.read_clickhouse_data manual__2025-10-02T11:34:10.329297+00:00 [queued]>
[2025-10-02T11:34:18.239+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.read_clickhouse_data manual__2025-10-02T11:34:10.329297+00:00 [queued]>
[2025-10-02T11:34:18.240+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-02T11:34:18.251+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): read_clickhouse_data> on 2025-10-02 11:34:10.329297+00:00
[2025-10-02T11:34:18.256+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3492) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T11:34:18.257+0000] {standard_task_runner.py:63} INFO - Started process 3493 to run task
[2025-10-02T11:34:18.258+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'main', 'read_clickhouse_data', 'manual__2025-10-02T11:34:10.329297+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpise635_9']
[2025-10-02T11:34:18.259+0000] {standard_task_runner.py:91} INFO - Job 78: Subtask read_clickhouse_data
[2025-10-02T11:34:18.294+0000] {task_command.py:426} INFO - Running <TaskInstance: main.read_clickhouse_data manual__2025-10-02T11:34:10.329297+00:00 [running]> on host 5adecdfd8c83
[2025-10-02T11:34:18.359+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='read_clickhouse_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T11:34:10.329297+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T11:34:10.329297+00:00'
[2025-10-02T11:34:18.360+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T11:34:18.371+0000] {sql.py:276} INFO - Executing: SELECT * FROM new_table
[2025-10-02T11:34:18.381+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-10-02T11:34:18.390+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-10-02T11:34:18.396+0000] {sql.py:487} INFO - Running statement: SELECT * FROM new_table, parameters: None
[2025-10-02T11:34:18.397+0000] {sql.py:496} INFO - Rows affected: 1
[2025-10-02T11:34:18.397+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T11:34:18.416+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=main, task_id=read_clickhouse_data, run_id=manual__2025-10-02T11:34:10.329297+00:00, execution_date=20251002T113410, start_date=20251002T113418, end_date=20251002T113418
[2025-10-02T11:34:18.431+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T11:34:18.445+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-02T11:34:18.448+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
